test/Hintman/Suggest.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Hintman.Suggest (tests) where
+
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+import Relude
+import qualified Data.Text as T
+
+data LineState = Plain | Trailing
+
+space :: MonadGen m => m Char
+space = pure ' '
+
+genSpaces :: Int -> Int -> Gen Text
+genSpaces i j = Gen.text (Range.linear i j) space
```

Comment:
Looks like you use this generator only once. In that case you can hardcode `1` and `20` numbers here instead of passing them as parameters.